Automattic/wp-calypso
# Reader: combine consecutive x-posts

Combine consecutive x-posts to the same post or comment.

Part of #14722.

![27597241-07d921ae-5b1f-11e7-8ca3-38cf57c97a65](https://user-images.githubusercontent.com/17325/42075015-0d3cc5d6-7bc3-11e8-8950-5a64c25e2886.png)

### To test

Visit a stream with x-posts (like the team stream). Make sure that consecutive x-posts for the same original post have been combined like this:

<img width="814" alt="screen shot 2018-07-04 at 15 08 58" src="https://user-images.githubusercontent.com/17325/42254735-4236ceee-7f9c-11e8-821f-8cbf2cee165b.png">
